python/fledge/services/core/api/plugins/remove.py

```diff
@@ -123,9 +128,10 @@ def purge_plugin(plugin_type: str, name: str):
         Remove plugin based on platform
     """
     _logger.info("Plugin removal started...")
+    org_name = name
```

Comment:
Do you mean here organization name?

Reply:
org_name = original_name

plugin Random
org_name = 'Random'
name = 'random'

I suggest to change name to original_name